core/server/data/migrations/versions/4.46/2022-04-13-12-58-fill-uuid-for-newsletters.js

```diff
@@ -0,0 +1,19 @@
+const logging = require('@tryghost/logging');
+const uuid = require('uuid');
+
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Setting missing uuid values for existing newsletters');
+        const uuidV4 = uuid.v4();
+        const updatedRows = await knex('newsletters')
+            .where('uuid', null)
+            .update('uuid', uuidV4);
```

Comment:
Doesn't this try to give all rows the same UUID, so it'll fail if there's more than one? 🤔

Reply:
Eugh yea good point, updating. I had in my mind only a single newsletter case here but we should definitely not rely on that.

Reply:
Updated.